.github/workflows/backward-compat-tests.yml

```diff
@@ -49,10 +49,17 @@ jobs:
         with:
           java-version: 1.8
       - name: Build
-        run: ./gradlew stream:server:build -x test ${GRADLE_ARGS}
+        run: mvn -B -nsu -am -pl :stream-storage-server clean install
       - name: Test current server with old clients
-        run: ./gradlew :tests:backward-compat:current-server-old-clients:test ${GRADLE_ARGS} || (tail -n +1 tests/backward-compat/current-server-old-clients/build/reports/tests/test/classes/* && tail -n +1 tests/backward-compat/current-server-old-clients/build/container-logs/**/* && exit 1)
+        run: mvn -B -nsu -am -pl :backward-compat-current-server-old-clients test
```

Comment:
-am is not needed if the maven phase is "test" 
In this case will run the tests for every dependant module ?

Reply:
@nicoloboschi `-am` only compile the module, it will not run tests. And it's needed, otherwise the test module can't compile success.